src/command/cmd_funcs.c

```diff
@@ -3899,11 +3899,19 @@ cmd_form(ProfWin *window, const char *const command, gchar **args)
     }
 
     if (g_strcmp0(args[0], "submit") == 0) {
-        iq_submit_room_config(confwin->roomjid, confwin->form);
+        if (confwin->submit == NULL) {
+            /* TODO */
```

Comment:
?

Reply:
Oups, just pushed some WIP commits. I was leaving this todo here because I was wondering if I should let the `wins_new_config` caller pass `NULL` as submit and cancel callbacks. And if so what was required to be done on call to `/form submit`. If you have any preferences please say so.